content/applications/finance/documents.rst

```diff
@@ -91,8 +106,88 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
+
+The scissors between pages indicate where the split will occur. Click on the scissors if you
+don't want to split your document between two specific pages.
+
+.. image:: documents/split-pdf.png
+   :align: center
+   :alt: split your documents
+
+.. note::
+
+   To merge documents, select them from your dashboard and click the :guilabel:`Split` icon. Then,
+   click on the scissors between pages to remove the cut, and click :guilabel:`Split`. Your
+   documents are now merged.
+
+Additional features
+-------------------
+
+Select a specific workspace and click on the arrow next to the :guilabel:`Upload` button to access
+additional features:
+
+Request
+~~~~~~~
+
+To request a missing file, click :guilabel:`Request`. Add the :guilabel:`Document name`, select the
+person you need it from in the :guilabel:`Request to` field, fill in the :guilabel:`Due date`, add
+the :guilabel:`Workspace` the document should belong to, and eventually add :guilabel:`Tags` and a
+:guilabel:`Note`. A file reflecting the missing document is created in the workspace.
+
+To see all the documents you requested, go to the :guilabel:`Activity` view and to the
+:guilabel:`Requested Document` column.
+
+.. Note::
+
+   To send a **reminder email** to all users you are waiting a document from, click on ⋮, and
+   :guilabel:`Document Request: Reminder`.
+
+   Click on a date to see the details of a specific request. You can update it by clicking on the
+   pen icon, :guilabel:`Preview` the content of the reminder email, or :guilabel:`Send Now` to send
+   a reminder email.
+
+   .. image:: documents/reminder-email.png
+      :align: center
+      :alt: send a reminder email from the Activity view
+
+Add a link
+~~~~~~~~~~
+
+To add a link to your documents dashboard, click on :guilabel:`Add a link`, enter the
+:guilabel:`URL`, and :guilabel:`Name` it. A file reflecting the url is created on your dashboard.
+
+Share
+~~~~~
+
+It is possible to share a document, a subworkspace, or an entire workspace with your contacts.
+
+To share the content of a workspace, select the workspace and click :guilabel:`Share`. A page pops
+up, with a :guilabel:`URL` available. You can :guilabel:`Copy` it to share it with the users you
+want so they can directly access your document workspace and its content. You can :guilabel:`Name`
+the share link and set a validity date for the url by filling the :guilabel:`Valid Until` field.
+Tick the :guilabel:`Include Sub Folders` box if you also want to share subfolders. You can also
+enable users to :guilabel:`Download` or :guilabel:`Download and Upload` documents.
+
+.. note::
+
+   It is possible to only share one or more documents (and not their workspace) by selecting them
+   and clicking :guilabel:`Share`.
+
+New spreadsheet
+~~~~~~~~~~~~~~~
+
+To create a new spreadsheet, click :guilabel:`New Spreadsheet`. You can select a
+:guilabel:`Blank spreadsheet` or an existing template.
+
+.. _documents/workflow-actions:
 
 Workflow actions
```

Comment:
Workflow actions help manage documents and overall business operations. These are automated actions that can be created and customized for each workspace. For example, create documents, process bills, sign, organize files, add tags to a file, or move it to another workspace with a single click etc. These workflow actions appear on the right panel when they meet the set criteria.